<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvMainWindow</class>
 <widget class="QMainWindow" name="vvMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vvResources.qrc">
    <normaloff>:/vvResources/Icons/logo.png</normaloff>:/vvResources/Icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClear_Menu"/>
    </widget>
    <widget class="QMenu" name="menuSaveAs">
     <property name="title">
      <string>Save &amp;As</string>
     </property>
     <addaction name="actionSavePCAP"/>
     <addaction name="actionSaveCSV"/>
     <addaction name="actionSavePositionCSV"/>
    </widget>
    <widget class="QMenu" name="menu_Open">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="actionOpenPcap"/>
     <addaction name="actionOpen_Sensor_Stream"/>
     <addaction name="separator"/>
     <addaction name="actionChoose_Calibration_File"/>
    </widget>
    <addaction name="menu_Open"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="menuSaveAs"/>
    <addaction name="actionExport_To_KiwiViewer"/>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuDebugging">
     <property name="title">
      <string>Debugging</string>
     </property>
     <addaction name="actionNative_File_Dialogs"/>
    </widget>
    <widget class="QMenu" name="menuDualReturnMode">
     <property name="title">
      <string>&amp;Dual Return Mode</string>
     </property>
     <addaction name="actionDualReturnModeDual"/>
     <addaction name="actionDualReturnDistanceNear"/>
     <addaction name="actionDualReturnDistanceFar"/>
     <addaction name="actionDualReturnIntensityHigh"/>
     <addaction name="actionDualReturnIntensityLow"/>
    </widget>
    <addaction name="actionSpreadsheet"/>
    <addaction name="actionMeasurement_Grid"/>
    <addaction name="actionGrid_Properties"/>
    <addaction name="actionLaserSelection"/>
    <addaction name="menuDualReturnMode"/>
    <addaction name="actionChoose_Calibration_File"/>
    <addaction name="actionCropReturns"/>
    <addaction name="separator"/>
    <addaction name="actionPlay"/>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionPython_Console"/>
    <addaction name="separator"/>
    <addaction name="menuDebugging"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_VeloView"/>
    <addaction name="separator"/>
    <addaction name="actionVeloViewUserGuide"/>
    <addaction name="actionVeloViewDeveloperGuide"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Basic Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenPcap"/>
   <addaction name="actionOpen_Sensor_Stream"/>
   <addaction name="actionChoose_Calibration_File"/>
   <addaction name="actionCropReturns"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_All_Points"/>
   <addaction name="actionSpreadsheet"/>
   <addaction name="actionOverheadView"/>
   <addaction name="actionPython_Console"/>
  </widget>
  <widget class="vvColorToolbar" name="colorToolBar">
   <property name="windowTitle">
    <string>Color Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="playbackToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGo_To_Start"/>
   <addaction name="actionSeek_Backward"/>
   <addaction name="actionPlay"/>
   <addaction name="actionSeek_Forward"/>
   <addaction name="actionGo_To_End"/>
   <addaction name="actionLoop"/>
   <addaction name="actionRecord"/>
  </widget>
  <widget class="QToolBar" name="viewSettings">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionReset_Camera"/>
   <addaction name="actionToggleProjection"/>
   <addaction name="actionMeasure"/>
   <addaction name="actionPlaneFit"/>
   <addaction name="separator"/>
   <addaction name="actionSetViewXPlus"/>
   <addaction name="actionSetViewXMinus"/>
   <addaction name="actionSetViewYPlus"/>
   <addaction name="actionSetViewYMinus"/>
   <addaction name="actionSetViewZPlus"/>
   <addaction name="actionSetViewZMinus"/>
  </widget>
  <widget class="QToolBar" name="geolocationToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="colorMapEditorDock">
   <property name="windowTitle">
    <string>Color Map Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqColorMapEditor" name="colorMapEditorPanel"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenPcap">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/WiresharkDoc-128.png</normaloff>:/VelodyneHDLPlugin/WiresharkDoc-128.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Capture File</string>
   </property>
   <property name="toolTip">
    <string>Open a Wireshark-format capture file with data collected from a Velodyne sensor</string>
   </property>
   <property name="statusTip">
    <string>Open Data File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelect_Visible_Points">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSurfaceSelectionPoint24.png</normaloff>:/vvResources/Icons/pqSurfaceSelectionPoint24.png</iconset>
   </property>
   <property name="text">
    <string>Select Visible Points</string>
   </property>
   <property name="toolTip">
    <string>Select Visible Points (s)</string>
   </property>
  </action>
  <action name="actionSelect_All_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqFrustumSelectionPoint24.png</normaloff>:/vvResources/Icons/pqFrustumSelectionPoint24.png</iconset>
   </property>
   <property name="text">
    <string>Select All Points</string>
   </property>
   <property name="toolTip">
    <string>Select All Points</string>
   </property>
  </action>
  <action name="actionSpreadsheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSpreadsheet16.png</normaloff>:/vvResources/Icons/pqSpreadsheet16.png</iconset>
   </property>
   <property name="text">
    <string>Spreadsheet</string>
   </property>
   <property name="toolTip">
    <string>Spreadsheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOpen_Sensor_Stream">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/velodyne_hdl.png</normaloff>:/VelodyneHDLPlugin/velodyne_hdl.png</iconset>
   </property>
   <property name="text">
    <string>Sensor &amp;Stream</string>
   </property>
  </action>
  <action name="actionChoose_Calibration_File">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/calibrate.png</normaloff>:/VelodyneHDLPlugin/calibrate.png</iconset>
   </property>
   <property name="text">
    <string>Choose &amp;Calibration File</string>
   </property>
  </action>
  <action name="actionSeek_Backward">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-seek-backward.png</normaloff>:/VelodyneHDLPlugin/media-seek-backward.png</iconset>
   </property>
   <property name="text">
    <string>Seek Backward</string>
   </property>
   <property name="toolTip">
    <string>Seek Backward</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-playback-start.png</normaloff>:/VelodyneHDLPlugin/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSeek_Forward">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-seek-forward.png</normaloff>:/VelodyneHDLPlugin/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>Seek Forward</string>
   </property>
   <property name="toolTip">
    <string>Seek Forward</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-playback-pause.png</normaloff>:/VelodyneHDLPlugin/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-record.png</normaloff>:/VelodyneHDLPlugin/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionPython_Console">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/python_logo_124.png</normaloff>:/vvResources/Icons/python_logo_124.png</iconset>
   </property>
   <property name="text">
    <string>Python Console</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqResetCamera32.png</normaloff>:/vvResources/Icons/pqResetCamera32.png</iconset>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close Data</string>
   </property>
   <property name="toolTip">
    <string>Close Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionMeasurement_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Measurement Grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGo_To_Start">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-skip-backward.png</normaloff>:/VelodyneHDLPlugin/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Go To Start</string>
   </property>
  </action>
  <action name="actionGo_To_End">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-skip-forward.png</normaloff>:/VelodyneHDLPlugin/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Go To End</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="text">
    <string>Save &amp;Screenshot</string>
   </property>
  </action>
  <action name="actionSaveCSV">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/csv.png</normaloff>:/VelodyneHDLPlugin/csv.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;CSV</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-loop.png</normaloff>:/VelodyneHDLPlugin/media-loop.png</iconset>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionExport_To_KiwiViewer">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/kiwi_200.png</normaloff>:/VelodyneHDLPlugin/kiwi_200.png</iconset>
   </property>
   <property name="text">
    <string>Export To &amp;KiwiViewer</string>
   </property>
   <property name="toolTip">
    <string>Export To KiwiViewer</string>
   </property>
  </action>
  <action name="actionGrid_Properties">
   <property name="text">
    <string>Grid Properties</string>
   </property>
  </action>
  <action name="actionSavePCAP">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/WiresharkDoc-128.png</normaloff>:/VelodyneHDLPlugin/WiresharkDoc-128.png</iconset>
   </property>
   <property name="text">
    <string>Save PCAP</string>
   </property>
  </action>
  <action name="actionAbout_VeloView">
   <property name="text">
    <string>About VeloView</string>
   </property>
  </action>
  <action name="actionVeloViewDeveloperGuide">
   <property name="text">
    <string>VeloView Developer Guide</string>
   </property>
  </action>
  <action name="actionRecent_file1">
   <property name="text">
    <string>recent_file1</string>
   </property>
  </action>
  <action name="actionRecent_file2">
   <property name="text">
    <string>recent_file2</string>
   </property>
  </action>
  <action name="actionRecent_file3">
   <property name="text">
    <string>recent_file3</string>
   </property>
  </action>
  <action name="actionRecent_file4">
   <property name="text">
    <string>recent_file4</string>
   </property>
  </action>
  <action name="actionRecent_file5">
   <property name="text">
    <string>recent_file5</string>
   </property>
  </action>
  <action name="actionClear_Menu">
   <property name="text">
    <string>Clear Menu</string>
   </property>
  </action>
  <action name="actionNative_File_Dialogs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Native File Dialogs</string>
   </property>
  </action>
  <action name="actionToggleProjection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/ViewOrtho.png</normaloff>
     <normalon>:/vvResources/Icons/ViewPerspective.png</normalon>:/vvResources/Icons/ViewOrtho.png</iconset>
   </property>
   <property name="text">
    <string>toggleProjection</string>
   </property>
   <property name="toolTip">
    <string>Toggle between projective and orthogonal view</string>
   </property>
  </action>
  <action name="actionSetViewXPlus">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqXPlus32.png</normaloff>:/vvResources/Icons/pqXPlus32.png</iconset>
   </property>
   <property name="text">
    <string>setViewXPlus</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to +X</string>
   </property>
  </action>
  <action name="actionSetViewXMinus">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqXMinus32.png</normaloff>:/vvResources/Icons/pqXMinus32.png</iconset>
   </property>
   <property name="text">
    <string>setViewXMinus</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to -X</string>
   </property>
  </action>
  <action name="actionSetViewYPlus">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqYPlus32.png</normaloff>:/vvResources/Icons/pqYPlus32.png</iconset>
   </property>
   <property name="text">
    <string>setViewYPlus</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to +Y</string>
   </property>
  </action>
  <action name="actionSetViewYMinus">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqYMinus32.png</normaloff>:/vvResources/Icons/pqYMinus32.png</iconset>
   </property>
   <property name="text">
    <string>setViewYMinus</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to -Y</string>
   </property>
  </action>
  <action name="actionSetViewZPlus">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqZPlus32.png</normaloff>:/vvResources/Icons/pqZPlus32.png</iconset>
   </property>
   <property name="text">
    <string>setViewZPlus</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to +Z</string>
   </property>
  </action>
  <action name="actionSetViewZMinus">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqZMinus32.png</normaloff>:/vvResources/Icons/pqZMinus32.png</iconset>
   </property>
   <property name="text">
    <string>setViewZMinus</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to -Z</string>
   </property>
  </action>
  <action name="actionVeloViewUserGuide">
   <property name="text">
    <string>VeloView User Guide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/Measurements.png</normaloff>:/vvResources/Icons/Measurements.png</iconset>
   </property>
   <property name="text">
    <string>Measure</string>
   </property>
   <property name="toolTip">
    <string>Measure the distance between two points: Press Control key and keep pressed left button of the mouse two place the two points of the ruler. To use this tool, the orthogonal projection view is obligatory.</string>
   </property>
  </action>
  <action name="actionOverheadView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/map.png</normaloff>:/vvResources/Icons/map.png</iconset>
   </property>
   <property name="text">
    <string>OverheadView</string>
   </property>
   <property name="toolTip">
    <string>Show overhead view of position</string>
   </property>
  </action>
  <action name="actionPlaneFit">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/planefit.png</normaloff>:/vvResources/Icons/planefit.png</iconset>
   </property>
   <property name="text">
    <string>Plane Fit</string>
   </property>
   <property name="toolTip">
    <string>Plane Fit to selection</string>
   </property>
  </action>
  <action name="actionSavePositionCSV">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSpreadsheet16.png</normaloff>:/vvResources/Icons/pqSpreadsheet16.png</iconset>
   </property>
   <property name="text">
    <string>Save Position CSV</string>
   </property>
  </action>
  <action name="actionGPSApply">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/location.png</normaloff>:/vvResources/Icons/location.png</iconset>
   </property>
   <property name="text">
    <string>GPSApply</string>
   </property>
   <property name="toolTip">
    <string>Apply geolocation</string>
   </property>
  </action>
  <action name="actionAbsolute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/absolute.png</normaloff>:/vvResources/Icons/absolute.png</iconset>
   </property>
   <property name="text">
    <string>absolute</string>
   </property>
   <property name="toolTip">
    <string>Geolocation in absolute coordinates</string>
   </property>
  </action>
  <action name="actionLaserSelection">
   <property name="text">
    <string>Laser Selection</string>
   </property>
  </action>
  <action name="actionSaveLAS">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/las.png</normaloff>:/VelodyneHDLPlugin/las.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;LAS</string>
   </property>
   <property name="toolTip">
    <string>Save to ASPRS LAS format</string>
   </property>
  </action>
  <action name="actionOpenApplanix">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/gps.png</normaloff>:/VelodyneHDLPlugin/gps.png</iconset>
   </property>
   <property name="text">
    <string>Capture File with External &amp;Geoposition</string>
   </property>
   <property name="toolTip">
    <string>Open a text file with POSPAC text-format position data collected from an Applanix sensor, and Wireshark-format capture file with data collected from a Velodyne sensor</string>
   </property>
  </action>
  <action name="actionCropReturns">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/crop.png</normaloff>:/VelodyneHDLPlugin/crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop Returns...</string>
   </property>
  </action>
  <action name="actionDualReturnModeDual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Dual</string>
   </property>
   <property name="toolTip">
    <string>Show both returns of a dual return</string>
   </property>
  </action>
  <action name="actionDualReturnDistanceNear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Near Distance</string>
   </property>
   <property name="toolTip">
    <string>Show nearer distance return of a dual return</string>
   </property>
  </action>
  <action name="actionDualReturnDistanceFar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Far Distance</string>
   </property>
   <property name="toolTip">
    <string>Show farther distance return of a dual return</string>
   </property>
  </action>
  <action name="actionDualReturnIntensityHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;High Intensity</string>
   </property>
   <property name="toolTip">
    <string>Show higher intensity return of a dual return</string>
   </property>
  </action>
  <action name="actionDualReturnIntensityLow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Low Intensity</string>
   </property>
   <property name="toolTip">
    <string>Show lower intensity return of a dual return</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vvColorToolbar</class>
   <extends>QToolBar</extends>
   <header>vvColorToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>pqColorMapEditor</class>
   <extends>QWidget</extends>
   <header>pqColorMapEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vvResources.qrc"/>
  <include location="VelodyneHDL/images/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>vvMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
